packages/tools/client-debugger/client-debugger/src/DevtoolsLogger.ts

```diff
@@ -151,22 +113,29 @@ export class DevtoolsLogger extends TelemetryLogger {
 	 * @param event - The telemetry event to send.
 	 */
 	public send(event: ITelemetryBaseEvent): void {
-		// TODO: ability to disable the logger so this becomes a no-op
-
-		const newEvent: ITimestampedTelemetryEvent = {
-			logContent: this.prepareEvent(event),
-			timestamp: Date.now(),
-		};
-
-		// insert log into the beginning of the array to show the latest log first
-		this._telemetryLog.unshift(newEvent);
-
-		// set log option to be undefined to avoid sending the log message to window console; these were too noisy
-		postMessagesToWindow(
-			undefined,
-			TelemetryEvent.createMessage({
-				event: newEvent,
-			}),
-		);
+		// Forward event to base logger
+		this.baseLogger?.send(event);
+
+		// TODO: ability to disable the logger so the rest of this becomes a no-op
+		try {
+			const newEvent: ITimestampedTelemetryEvent = {
+				logContent: this.prepareEvent(event),
+				timestamp: Date.now(),
+			};
+
+			// insert log into the beginning of the array to show the latest log first
+			this._telemetryLog.unshift(newEvent);
```

Comment:
@alexvy86 Not for this PR, but is there a reason for us to keep this order inverted here? Presumably the consuming view could just sort the messages by timestamp to get the appropriate ordering.

Reply:
Having to sort on the view on every update feels a bit wasteful given how frequent telemetry events can be. But since we're limiting to 1000 events tops if I recall correctly, it's not the end of the world if we do it. Do you see any issues with the current approach, or just for simplicity in the logger?

Reply:
It's mostly about logger simplicity. But it also just feels odd from an API perspective to invert the events - in my mind, the logger shouldn't really know anything about how the contents it broadcasts ultimately get displayed. I'm assuming we will eventually support toggling different sort patterns in the view as well, in which case this stops being all that useful.